src/Features/Core/Portable/InlineHints/InlineHint.cs

```diff
@@ -16,20 +16,22 @@ namespace Microsoft.CodeAnalysis.InlineHints
         public readonly TextSpan Span;
         public readonly ImmutableArray<TaggedText> DisplayParts;
         public readonly TextChange? ReplacementTextChange;
+        public readonly double Ranking;
         private readonly Func<Document, CancellationToken, Task<ImmutableArray<TaggedText>>>? _getDescriptionAsync;
 
         public InlineHint(
             TextSpan span,
             ImmutableArray<TaggedText> displayParts,
             Func<Document, CancellationToken, Task<ImmutableArray<TaggedText>>>? getDescriptionAsync = null)
-            : this(span, displayParts, replacementTextChange: null, getDescriptionAsync)
+            : this(span, displayParts, replacementTextChange: null, ranking: 0.0, getDescriptionAsync)
         {
         }
 
         public InlineHint(
             TextSpan span,
             ImmutableArray<TaggedText> displayParts,
             TextChange? replacementTextChange,
+            double ranking,
             Func<Document, CancellationToken, Task<ImmutableArray<TaggedText>>>? getDescriptionAsync = null)
```

Comment:
will this break our partners?  do they use these apis?  do they use EA for them?

Reply:
I do not believe anyone does, but I will just make a separate constructor to be safe.